lib/resque/durable/background_heartbeat.rb

```diff
@@ -0,0 +1,83 @@
+require 'thread'
+
+module Resque
+  module Durable
+
+    # Creates a background thread to regularly heartbeat the queue audit.
+    class BackgroundHeartbeat
+      DEFAULT_INTERVAL = 15
```

Comment:
This is vestigial. Since we're opt-in (right now), you have to explicitly set a value. I'll remove.